app/models/in_progress_form.rb

```diff
@@ -28,7 +28,9 @@ def cast(value)
                                }
   # the double quotes in return_url are part of the value
   scope :return_url, ->(url) { where(%( #{RETURN_URL_SQL} = ? ), "\"#{url}\"") }
-
+  scope :is_fsr, -> { where(form_id: '5655') }
```

Comment:
I think it'd be nice to expand some of these scopes to be flexible for any `form_id` whether that's mapping over the forms with each scope (just what I threw together quickly probably a cleaner way to approach it)

```
FormProfile::FORM_ID_TO_CLASS.keys.each do |form_id|
  scope "is_#{form_id}".underscore.to_sym, where(form_id:)
end
```

Or providing an argument

`scope :for_form, ->(form_id) { where(form_id:) }`

Just in case another team might have a use case for this scope it doesn't clutter the model

Reply:
Changes made